protocol/avdl/keybase1/wot.avdl

```diff
@@ -0,0 +1,23 @@
+@namespace("keybase.1")
+
+protocol wot {
+  import idl "common.avdl";
+
+  enum UsernameVerificationType {
+    NONE_0,
+    AUDIO_1,
+    VIDEO_2,
+    EMAIL_3,
+    OTHER_CHAT_4,
+    IN_PERSON_5
+  }
+
+  record Confidence {
```

Comment:
if possible can we try to do everything in terms of one common data structure? The more transmutation code we can avoid, the better.

Reply:
We need something in avdl so that frontend can use it.  I can try to make it so this matches what the server needs with @jsonkey.

In the future, I think starting with the avdl would make things easier than starting with api server as avdl should be what ties it all together.

Reply:
Agreed about the `@jsonkey`. I'm pretty sensitive to this nowadays when designing the server, so let me know how I can change the server to help you

Reply:
I think transmuting will be required as we try to make it easy for frontend, and they can't fill out the proofs section easily to satisfy the server.  If the server took sig_ids instead, that could be easier, but service would still need to fill in the state.